### PR TITLE
feat: JSON schema builder update

### PR DESCRIPTION
**Pull request by bbtgnn**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -86,6 +86,7 @@ setup_frontend: $(WEBAPP)/.env
 	cd $(WEBAPP) && pnpm i
 
 setup: $(AZC) $(WZC) $(ZENROOM) $(ZENCODE) $(PB) setup_frontend ## 📦 Setup the project
+setup_arm: $(AZC) $(WZC) $(PB) setup_frontend
```
Comment (puria) [inline Makefile:89]: If you need different architectures don't change the logic to be system wide... Instead add specific logic to be consistent and have (local installed) zenroom version with your architecture. I'm sure you'll find online plenty of examples for the os arch differentiation on makefiles, maybe also on other repos like starters

File: admin/pb_migrations/1708362089_updated_templates.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/// <reference path="../pb_data/types.d.ts" />
+migrate((db) => {
+  const dao = new Dao(db)
+  const collection = dao.findCollectionByNameOrId("7amth7pix3i8k5p")
+
+  // remove
+  collection.schema.removeField("xozvdofz")
```
Comment (puria) [inline admin/pb_migrations/1708362089_updated_templates.js:7]: Should we normalize these migrations?

File: webapp/src/routes/my/organizations/[id]/services/_partials/JSONSchemaInput.svelte [hunk 3]
```diff
@@ -1,49 +1,24 @@
...
-	import { FieldWrapper } from '$lib/forms';
-	import { getFormContext } from '$lib/forms/form.svelte';
-	import {
-		JSONSchemaField,
-		JSONSchemaEditor,
-		type JSONSchemaInput
-	} from 'json-schema-builder-svelte';
-	import { Alert, TabItem, Tabs } from 'flowbite-svelte';
-	import { nanoid } from 'nanoid';
+	import { JSONSchemaEditor } from 'json-schema-builder-svelte';
```
Comment (puria) [inline webapp/src/routes/my/organizations/[id]/services/_partials/JSONSchemaInput.svelte:3]: 🎊

File: webapp/src/routes/my/organizations/[id]/services/_partials/JSONSchemaInput.svelte [hunk 4]
```diff
@@ -1,49 +1,24 @@
...
 	export let superform: SuperForm<UnwrapEffects<T>, any>;
 
 	const { value } = formFieldProxy(superform, field);
-
-	const openEditor = $value === '';
-	const openPlain = $value !== '';
-
-	let initialSchemaInput: Partial<JSONSchemaInput> = {};
-	if (mode == 'create') initialSchemaInput = { $id: `didroom-schema-${nanoid(5)}` };
+	console.log($value);
```
Comment (puria) [inline webapp/src/routes/my/organizations/[id]/services/_partials/JSONSchemaInput.svelte:13]: Is this necessaire?

File: admin/pb_migrations/1708362089_updated_templates.js (hunk 2 above)
Comment (bbtgnn) [reply admin/pb_migrations/1708362089_updated_templates.js:7]: maybe with a little script?

File: Makefile (hunk 1 above)
Comment (bbtgnn) [reply Makefile:89]: I'll remove for now, and make an issue on starters

File: admin/pb_migrations/1708362089_updated_templates.js (hunk 2 above)
Comment (bbtgnn) [reply admin/pb_migrations/1708362089_updated_templates.js:7]: Also here, we should make an issue on starters about this

**Pull request merged**